### PR TITLE
chore: update php sdk version

### PR DESCRIPTION
**Pull request by rishtigupta**
## PR Description:
- Update PHP SDK Version to the latest "v.1.7.1"
- Used a dockerfile to update the composer.lock to be in sync with composer.json

## Issue
https://github.com/momentohq/dev-eco-issue-tracker/issues/719

### CODE REVIEW

**Review by cprice404 — COMMENTED** [review]
is this still a draft or is it ready for review

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "source": "https://github.com/momentohq/laravel-cache"
   },
   "require": {
-    "momentohq/client-sdk-php": "1.3.0"
+    "momentohq/client-sdk-php": "v1.7.1"
```
Comment (cprice404) [inline composer.json:25]: i've been confused as to why these dep versions are switching between not having a `v` prefix and having a `v` prefix. do you know why?

Comment (rishtigupta) [reply composer.json:25]: I think git prefers using v for version number from now on: https://www.flownative.com/en/blog/semantic-versioning-and-different-version-number-formats.html#:~:text=Git,common%20way%20to%20denote%20versions.

Comment (cprice404) [reply composer.json:25]: so does it support both, and we are just standardizing on the one that we think is now "best practice"? or did something change about our release process such that it used to require no `v` and now requires `v`?

**Review by cprice404 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (rishtigupta) [reply composer.json:25]: It supports both, the composer does not break if we do not provide a 'v' prefix to our version number.

Comment (rishtigupta) [reply composer.json:25]: It supports both, the composer does not break if we do not provide a 'v' prefix to our version number.

Comment (cprice404) [reply composer.json:25]: got it. thank you!

**Pull request merged**